ground-core/pom.xml

```diff
@@ -182,6 +182,27 @@
             <version>${dropwizard.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>0.10.0.1</version>
```

Comment:
Can we pull version numbers out and put them in the root `pom.xml` and use variables? That way, we can enforce version uniformity across the project.